### ISSUE TITLE
color comes late

### ISSUE THREAD

**Issue by krux02** [syohex/emacs-quickrun#86]
When I run quickrun, it puts all output to a buffer, and applies colors after it is finished. In my opinion that's a bit ugly, especially when you are watching what is happening in that buffer: 

```
[32mHint: [0mhello_triangle[36m [Processing]
[0m[32mHint: [0mmacros[36m [Processing]
[0m[32mHint: [0msequtils[36m [Processing]
[0m[32mHint: [0mx[36m [Processing]
[0m[32mHint: [0mxlib[36m [Processing]
[0m[32mHint: [0mxutil[36m [Processing]
[0m[32mHint: [0mkeysym[36m [Processing]
[0m[32mHint: [0mdynlib[36m [Processing]
[0m[32mHint: [0mstrutils[36m [Processing]
[0m[32mHint: [0mparseutils[36m [Processing]
[0m[32mHint: [0mmath[36m [Processing]
[0m[32mHint: [0malgorithm[36m [Processing]
[0m[32mHint: [0mvec[36m [Processing]
[0m[32mHint: [0mmat[36m [Processing]
[0m[32mHint: [0mmat_transform[36m [Processing]
[0m[32mHint: [0mglobals[36m [Processing]
[0m[32mHint: [0mquat[36m [Processing]
[0m[32mHint: [0mtimes[36m [Processing]
[0m[32mHint: [0munsigned[36m [Processing]
[0m[32mHint: [0mposix[36m [Processing]
[0m[32mHint: [0mospaths[36m [Processing]
```

When I use quickrun-shell, all color is applied directly, but this one has the disadvantage, that it does change focus to that window, even though I do not want that (I configured quickrun-focus-p). I almost wanted to commit the issue, that quickrun-focus-p does not have an effect on `quickrun-shell`, but then I realized it was intentional, because that command is interactive programs. Is there a solution for my problem?

**Comment by syohex** [comment]
I think color issue is fixed by #89.

> I almost wanted to commit the issue, that quickrun-focus-p does not have an effect on quickrun-shell, but then I realized it was intentional, because that command is interactive programs. Is there a solution for my problem?

I'll fix later.

**Comment by syohex** [comment]
Window focus issue is fixed by #90. Please check latest version.
